config/initializers/apipie.rb

```diff
@@ -15,6 +15,7 @@
   config.default_locale = FastGettext.default_locale
   config.locale = ->(loc) { loc ? FastGettext.set_locale(loc) : FastGettext.locale }
 
+  ActiveRecord::Migration.maintain_test_schema!
```

Comment:
Ah, fair enough.



> In any case maybe a good solution is to call maintain_test_schema in a separate, earlier initializer?

Indeed, it's in a surprising place here.

If it was called only `if Rails.env.test?` then it may avoid the need to provide db/schema.rb too.